Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -1206,7 +1206,7 @@
       - NotekeeperCartridge
       - NewsReaderCartridge
       - CrimeAssistCartridge
-      - SecWatchCartridge
+      - LogProbeCartridge
```

Comment:
Do not remove SecWatch

Reply:
there is presently a bug that doesn't allow logprobe to be installed